README.md

```diff
@@ -115,6 +115,8 @@ The following `index` options are supported:
 - `length` (integer or hash) - The partial index length(s). If an integer is provided, it is used as the length for all columns. If a hash is provided, it is used to specify the length for individual columns, where the column names are given as `Symbol` hash keys.
 - `where` (string) - The subset index predicate.
 
+**Note:** The `DeclareSchema.max_index_and_constraint_name_length` setting is ignored when using `PostgreSQL` since that database does not have a limit on the length of index names, and requires globally unique index names.
+
```

Comment:
How about adding an explicit section for "Tested Databases"? That would list that it's tested with:
- Sqlite
- Postgres
- MySQL

It might be cool to even mention the major versions we're testing with, like 5.7 and 8.0 for MySQL.